app.ts

```diff
@@ -31,5 +30,8 @@ app.listen(3000, ()=> {
 });
 
 app.get('/', (req, res) => {
-  res.send('hello world')
-})
+  res.render('index')
+});
+app.get('/academy', (req, res) => {
+  res.render('academy')
+});
```

Comment:
this doesn't look necessary

Reply:
needed to access front end page as its not GETting anything from API